pkg/runctl/run_manager.go

```diff
@@ -278,6 +283,35 @@ func (c *runManager) copyPipelineCloneSecretToRunNamespace(ctx *runContext, secr
 	return names[0], nil
 }
 
+func (c *runManager) copyElasticSearchCredentialToRunNamespace(ctx *runContext, secretHelper secrets.SecretHelper) (string, error) {
+	if ctx.pipelineRun.GetSpec().Logging.Elasticsearch.ElasticSearchCredential == "" {
+		return "", nil
+	}
+
+	secretName := ctx.pipelineRun.GetSpec().Logging.Elasticsearch.ElasticSearchCredential
+	if secretName == "" {
+		return "", nil
+	}
+	repoServerURL, err := ctx.pipelineRun.GetPipelineRepoServerURL()
+	if err != nil {
+		// TODO: this method should not modify the pipeline run -> must be handled elsewhere
+		ctx.pipelineRun.UpdateMessage(err.Error())
+		ctx.pipelineRun.UpdateResult(v1alpha1.ResultErrorContent)
+		return "", err
+	}
+	transformers := []secrets.SecretTransformer{
+		secrets.StripAnnotationsTransformer("jenkins.io/"),
+		secrets.StripLabelsTransformer("jenkins.io/"),
+		secrets.UniqueNameTransformer(),
+		secrets.SetAnnotationTransformer("tekton.dev/git-0", repoServerURL),
```

Comment:
```suggestion
```
This is not required as this is no secret to clone from a git repository if I got it right.